ui/projects/Search.qml

```diff
@@ -0,0 +1,43 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+
+import "../../backend/core"
+
+
+Item {
+    id: root
+
+    height: combobox.height
+
+    property alias index: combobox.currentIndex
+
+    ComboBox {
+        id: combobox
+
+        width: root.width
+        model: Store.projects_project_settings.projectObj.projects
+
+        textRole: "name"
+
+        editable: true
+
+        onAccepted: {
+            if (find(editText) === -1) {
+                console.log("nothing to do")
+            }
+        }
+
+        onCurrentIndexChanged: {
+            Store.processing = true;
+            Store.processingChanged();
```

Comment:
No needs to call this function or it will be called twice as it is implicitly called when you are doing Store.processing = true